redaxo/src/addons/install/lib/api/api_core_update.php

```diff
@@ -25,6 +25,8 @@ public function execute()
         $installAddon = rex_addon::get('install');
         $versions = self::getVersions();
         $versionId = rex_request('version_id', 'int');
+        $newCoreVersion = rex_request('version', 'string');
```

Comment:
In Zeile 33 (vorher 31) wird die Version per `$version = $versions[$versionId];` geholt. Die braucht also nicht separat per Parameter übergeben werden.